R/db.R

```diff
@@ -68,7 +68,12 @@ mintr_db <- R6::R6Class(
         ret[[v]] <- round(ret[[v]] * options$population)
       }
       table <- mintr_db_transform_metabolic(ret, options$metabolic)
-      mintr_db_set_not_applicable_values(table)
+      # Sort by sensible intervention ordering
+      table$intervention <- factor(table$intervention,
+                                    levels = c("none", "llin", "llin-pbo", "pyrrole-pbo", "irs", "irs-llin", "irs-llin-pbo", "irs-pyrrole-pbo"))
+      sorted <- table[order(table$intervention),]
+      mintr_db_set_not_applicable_values(sorted)
```

Comment:
Try this, a bit more explicit:

```suggestion
     ordering <- c("none", "llin", "llin-pbo", "pyrrole-pbo", "irs", "irs-llin", "irs-llin-pbo", "irs-pyrrole-pbo")
     sorted  <- table[order(match(table$intervention, ordering)), ]
     mintr_db_set_not_applicable_values(sorted)
```